azure-pipelines.yml

```diff
@@ -4,7 +4,7 @@ jobs:
 - template: ci/azure/posix.yml
   parameters:
     name: macOS
-    vmImage: xcode9-macos10.13
+    vmImage: macos10.15
```

Comment:
FWIW I would be OK with `macos-latest` - I don't see a need to pin a particular version

Reply:
But I think failure is being 10.15 isn't available via pipelines:

https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent

Reply:
> FWIW I would be OK with `macos-latest` - I don't see a need to pin a particular version

Though so as well, but I figured that ```xcode9-macos10.13``` is there for a reason, apparently I was wrong

Reply:
Using latest is a little dicey IMO.

You don't know what incompatibilities newer versions may introduce.

Reply:
Using latest is a little dicey IMO.

You don't know what incompatibilities newer versions may (unexpectedly) introduce. I think pinning to the latest (numeric) version as was done before is best.

I don't necessarily object to testing against macos-latest. However, I would mark that build as an allowable failure (though not sure how one does that with Azure pipelines).